src/commands/ml-models-list.ts

```diff
@@ -0,0 +1,12 @@
+import { Command } from "../command";
+import { FirebaseModel } from "../ml/models";
+
+module.exports = new Command("ml:models:list")
```

Comment:
prefer `export default` in this case
```suggestion
export default new Command("ml:models:list")
```